Seminars/Sem.03/Pract.03/Task02.cpp

```diff
@@ -0,0 +1,166 @@
+#include <iostream>
+#include <fstream>
+
+using std::cout;
+using std::cin;
+using std::endl;
+using std::ios;
+using std::ofstream;
+using std::ifstream;
+
+const int MAX_COMPANY_NAME_LENGTH = 25;
+const int MAX_INPUT_LENGTH = 256;
+
+struct jobOffer {
+	char companyName[MAX_COMPANY_NAME_LENGTH]; //25

```

Comment:
+1 for null terminator

Reply:
int main() {
	char name[25];
	cout << sizeof(name); //prints 25
}
It is 25 I checked. 

There are 3 bytes padding after the last element of the companyName variable followed by 4 bytes for the teamCount variable.